<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Ground segmentation -->
  <node name="ground_segmentation_test" pkg="linefit_ground_segmentation_ros" type="ground_segmentation_test_node" output="screen">
    <rosparam command="load" file="$(find linefit_ground_segmentation_ros)/launch/segmentation_params.yaml"/>
    <param name="point_cloud_file" value="$(find linefit_ground_segmentation_ros)/../doc/kitti.ply" />
    <param name="visualize" value="true" />
  </node>

</launch>
